web/templates/user_stream_list_item.hbs

```diff
@@ -1,10 +1,10 @@
 <tr data-stream-id="{{stream_id}}">
-    <td class="channel_container">
+    <td class="channel-container">
```

Comment:
Again, these changes belong with your previous commit. Introduce the hyphens there.

Reply:
Fixed this myself and pushed. Marking for integration review. Thanks, @userAdityaa!

Reply:
And then this junk in the second commit:

``` diff
diff --git a/web/styles/popovers.css b/web/styles/popovers.css
index 27992b9d32..18cddb9dda 100644
--- a/web/styles/popovers.css
+++ b/web/styles/popovers.css
@@ -641,11 +641,11 @@ ul.popover-group-menu-member-list {
                         padding-right: 10px;
                     }
 
-                    &.channel-container {
+                    &.user-profile-channel-container {
                         text-align: left;
                         padding-left: 10px;
 
-                        & .subscription_list_stream {
+                        & .user-profile-channel-list {
                             display: flex;
                             gap: 3px;
                         }
```